diaphora.py

```diff
@@ -598,7 +598,11 @@ def save_function(self, props):
         for bb in bb_relations[key]:
           bb = str(bb)
           key = str(key)
-          cur_execute(sql, (bb_ids[key], bb_ids[bb]))
+          try:
```

Comment:
This is wrong. We should not be ignoring errors because there is no relationship found by Radare.